app/views/components/pickadate.html

```diff
@@ -4,3 +4,11 @@
   </span>
   <input type="text" placeholder="Select a date..." class="form-control" ng-disabled="disabled">
 </div>
+<div ng-if="minDate || maxDate" class="help-block row">
```

Comment:
Since you are using `row` here, the content needs to be in a column. Probably `col-sm-12` in this case. See http://getbootstrap.com/css/#grid